src/main/java/com/openlattice/chronicle/services/twilio/TwilioService.kt

```diff
@@ -26,13 +29,21 @@ class TwilioService(configuration: TwilioConfiguration) :
 
     private val fromPhoneNumber: PhoneNumber = PhoneNumber(configuration.fromPhone)
 
-    private fun sendMessage(messageDetails: MessageDetails) :MessageOutcome {
+    private fun sendMessage(organizationId: UUID, messageDetails: MessageDetails) :MessageOutcome {
         val messageText = "Chronicle device enrollment:  Please download app from your app store and click on ${messageDetails.url} to enroll your device."
         try {
             val message = Message
                 .creator(PhoneNumber(messageDetails.phoneNumber), fromPhoneNumber, messageText)
-                .setStatusCallback(URI.create("https://api.openlattice.com/bifrost/messages/status"))
+                .setStatusCallback(URI.create(
+                    "https://api.openlattice.com$BASE/$organizationId$MESSAGE_PATH$STATUS_PATH"))
                 .create()
+            logger.info(
+                """
+                    message sent to participant ${messageDetails.participantId}
+                    for study ${messageDetails.studyId}
+                    in organization $organizationId
+                """.trimIndent()
+            )
```

Comment:
No change required. This will log a multi-line string which makes it harder to grep for, and possibly not what you intended.